source/html/renderer/src/card-elements.ts

```diff
@@ -2431,7 +2458,7 @@ function raiseExecuteActionEvent(action: ExternalAction) {
     }
 }
 
-function raiseInlineCardExpandedEvent(action: ShowCardAction, isExpanded: boolean) {
+function raiseInlineCardExpandedEvent(action: ShowCardAction | HttpAction, isExpanded: boolean) {
```

Comment:
For now, please simple revert to just accepting ShowCardAction. This event is used by the WebChat control so it can scroll the card into view when its layout changes. The WebChat control will not use setStatus. If necessary, we will add a more generic onCardLayoutChanged event (or maybe even replace onInlineCardExpanded with onCardLayoutChanged) but we don't need to do it now.